include/ring-core/target.h

```diff
@@ -23,23 +23,13 @@
 // instead.
 
 #if defined(__x86_64) || defined(_M_AMD64) || defined(_M_X64)
-#define OPENSSL_64_BIT
```

Comment:
I realize now that some projects will break if we rely on build.rs to define OPENSSL_64_BIT and OPENSSL_32_BIT because some projects aren't using build.rs. I suggest an alternative in PR #2082. PTAL and let me know if that works for you.

Reply:
Thanks about that. However, I think it won't work for the arch csky which is using gnu6 now. 

How about we define it in both build.rs and target.h?